containers/cuda/Containerfile

```diff
@@ -1,20 +1,91 @@
 # SPDX-License-Identifier: Apache-2.0
+#
+# This container file requires a RHEL subscription to build and
```

Comment:
this is a good note to add.

@tiran do you think this is a barrier and we should use the cuda ubuntu img instead?

Reply:
Actually I was wrong and could build the container without an active subscription. I must have confused this container with another container that pulls in dependencies that are not part of public UBI9.

I'm removing the comment again.